source/_references/usermetadata.md

```diff
@@ -0,0 +1,113 @@
+---
+title: User metadata
+use: [references]
+---
+
+## Intro
+
+User metadata is metadata added to a source image by the API-user that doesn't change the image identifying hash.
+
+## User metadata types
+
+User metadata can be a string, integer, double, date, latitude/longitude or an array and also be changed after a image was uploaded.
+
+Default is string, if you want to store it as any other type, you have to specify it with a prefix
+
+| Type     | Prefix  |
+-----------|----------
+| Int      | int:    |
+| Double   | double: |
+| Date     | date:   |
+| Location | latlon: |
+| Array    | array:  |
+
+
+Date needs to be in the _YYYY-MM-DDThh:mm:ssZ_ format. Location needs to be a latitude and longitude in the following form, eg: "47.38,8.52".
+The elements of an array will always be stored as string, you can't specify a type there.
+
+## Add user metadata to a source image
+
+To add user metadata to a source image, you need to provide the organization, the identifying hash of the image, the name of the user metadata and the values to add.
+
+Regular index field names must begin with a letter and be at least 3 and no more than 54 characters long. The allowed characters are: a-z (lower-case letters), 0-9, and _ (underscore).
+
+The most flexible way to add/replace/delete fields is with a PATCH request and having the field data JSON-encoded in the body.
+This also allows to update more than one field at once.
+
+```bash
+curl -H 'Content-Type: application/json' -X PATCH 'https://api.rokka.io/sourceimages/testorganization/0dcabb778d58d07ccd48b5ff291de05ba4374fb9/meta/user' -d '{
+    "somefield": "somevalue",
+    "int:someNumber": 0,
+    "date:someDate": "2017-01-29T12:34:56Z",
+    "array:someArray": ["Foo","Bar","Baz"],
+    "deleteThis": null
+    }'
+```
+
+
+```php
+$client = \Rokka\Client\Factory::getImageClient('testorganization', 'apiKey', 'apiSecret');
+
+$hash = '0dcabb778d58d07ccd48b5ff291de05ba4374fb9';
+
+$client->addUserMetadata([ "somefield" => "somevalue",
+                             "somethingElse" => "anothervalue",
+                             "date:someFate" => new \DateTime("2017-01-29T12:34:56Z"),
+                             "deleteThis" => null
+                             ], $hash);
+
+```
+If you do a PUT (or setUserMetadata in PHP) instead of a PATCH request, then all existing fields will be deleted first.
```

Comment:
remove `(or setUserMetadata in PHP)`, that's API client specific and has no place here, IMHO.